src/app/exercise/exercise/exercise.base.ts

```diff
@@ -72,7 +72,8 @@ export class ExerciseBase implements OnDestroy {
   constructor(public slide: SlideComponent, private monacoConfig: MonacoConfigService) {
     this.onActiveUsubscribe = slide.onActive.filter(a => a).subscribe(() => {
       console.log('ACTIVE');
-      slide.disableShortcuts();
+
+      // slide.disableShortcuts();
```

Comment:
ideally not just remove the call, but get rid of the any mention of disableShortcuts/enableShortcuts and areShortcuts enabled at all.
Your solution is so much better!

If that sounds too scary, lmk, and I can do it myself